tests/iid_example.py

```diff
@@ -0,0 +1,67 @@
+import sys
+
+sys.path.append("..")
+from decanter_ai_sdk.client import Client
+import os
+from decanter_ai_sdk.enums.evaluators import ClassificationMetric
+from decanter_ai_sdk.enums.data_types import DataType
+
+
+def test_iid():
+    print("---From test iid---")
+
+    client = Client(
+        auth_key="auth_key",
```

Comment:
make an indication that users should fill in these values themselves. For example

```
   auth_key = "" #TODO fill in real authorization key"
   ...
   client = Client(
        auth_key=auth_key,
        ...
    )
```